website/content/en/docs/advanced-topics/custom-bundle-validation.md

```diff
@@ -5,18 +5,20 @@ weight: 80
 
 ## Summary
 
-Operator Authors can now use "external" validators with the
+Operator authors can now use "external" validators with the
 `operator-sdk bundle validate` command by using the
-`--alpha-select-external` flag. This feature enable Operator authors,
-users, and registry pipelines to use custom validators.
+`--alpha-select-external` flag. This feature enables Operator authors,
+users, and registry pipelines to use custom validators. These custom
+validators can be written in any language. 
 
 ## Usage
 
 External validators can be used by specifying a list of local filepaths to
 executables using colons as path separators:
 
 ```sh
-operator-sdk bundle validate --alpha-select-external path/validator1:path/validator2
+$ operator-sdk bundle validate --alpha-select-external \ 
+path/validator1:path/validator2
```

Comment:
Should we move the flag down with the flag arguments for better readability?

Something like:
```suggestion
$ operator-sdk bundle validate \ 
--alpha-select-external path/validator1:path/validator2
```